backend/src/db/sqlite.py

```diff
@@ -17,13 +18,37 @@
 else:
     AIOSQLITE_IMPORT_ERROR = None
 
+class SQLiteColumnDefinition(SQL_column_definition):
+
+    type_map = {
+        int: "INTEGER",
+        float: "REAL",
+        str: "TEXT",
+        datetime.datetime: "TEXT"
+    }
+
+    def __init__(self, name:str, data_type:type, constraint:str=None):
+        if type in self.type_map:
+            self.data_type = self.type_map[type]
+            self.name = name
+        else:
+            raise ValueError(f"Unsupported data type for an SQLite Column definition: {type}")
+
+class SQLiteSQLFactory(SQLFactory):
+
+    def getClass(self, sql_cls: type):
+        if sql_cls.__name__ == "SQL_column_definition":
```

Comment:
Namen Attribut der Basisklasse verwenden: `sql_cls.__name__ in [b.__name__ for b in SQLiteColumnDefinition.__bases__ ]`